content/en/_index.md

```diff
@@ -197,6 +236,26 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="C#" >}}
+public static ListInfoTypesResponse ListInfoTypes(string languageCode, string filter)
+{
+    var dlp = DlpServiceClient.Create();
```

Comment:
Should we be showing how to use the `using` statement here as described delow?

Should it have the comment clarifying usage as well?

Reply:
Updated.

Reply:
Rolled back update based on feedback from amanda-tarafa@

Reply:
Clarifying comments added.